cluster_agent/identity/slurm_user/mappers/ldap.py

```diff
@@ -12,6 +13,8 @@
 
 set_library_log_detail_level(ERROR)
 
+TIMEOUT_SECONDS = 30
```

Comment:
so, I believe this constant should be placed inside the LDAPMapper class as a internal variable. The way as is, it is accessible by the other modules but has no utility at all. Even if I couldn't easily think on a use case that will change the value of this variable and lead to unwanted behaviours, I think it is a bit dangerous. What do you think?

Reply:
solved in [0a59023](https://github.com/omnivector-solutions/cluster-agent/pull/68/commits/0a59023ce8021401e4a5b0695cf1d9c16615add0)